com.civicrm.hrjobroles/js/test/controllers/job-roles.controller.spec.js

```diff
@@ -59,89 +59,44 @@ define([
     });
 
     describe('getOptionValues', function () {
+      var expectedDepartmentValue, expectedLocationValue, expectedLevelValue, expectedRegionValue;
+
       beforeEach(function () {
-        spyOn(jobRoleService, 'getOptionValues').and.returnValue($q.resolve({
-          'count': 5,
-          'values': [
-            {
-              'id': '845',
-              'option_group_id': '111',
-              'label': 'Senior Manager',
-              'value': 'Senior Manager',
-              'name': 'Senior_Manager',
-              'is_default': '0',
-              'weight': '1',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '846',
-              'option_group_id': '111',
-              'label': 'Junior Manager',
-              'value': 'Junior Manager',
-              'name': 'Junior_Manager',
-              'is_default': '0',
-              'weight': '2',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '879',
-              'option_group_id': '124',
-              'label': 'Other',
-              'value': 'Other',
-              'name': 'Other',
-              'filter': '0',
-              'is_default': '0',
-              'weight': '3',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '1045',
-              'option_group_id': '124',
-              'label': 'Test A',
-              'value': '1',
-              'name': 'Test A',
-              'filter': '0',
-              'is_default': '0',
-              'weight': '2',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            },
-            {
-              'id': '1046',
-              'option_group_id': '124',
-              'label': 'Test B',
-              'value': '2',
-              'name': 'Test B',
-              'filter': '0',
-              'is_default': '0',
-              'weight': '1',
-              'description': 'Test B',
-              'is_optgroup': '0',
-              'is_reserved': '0',
-              'is_active': '1'
-            }
-          ],
-          'optionGroupData': {
-            'cost_centres': '124'
-          }
-        }));
+        spyOn(jobRoleService, 'getOptionValues').and.returnValue($q.resolve(Mock.option_values));
         initController();
         $rootScope.$digest();
+        expectedLevelValue = Mock.option_values.values[0].value;
```

Comment:
Please move assignments to the top of the function, then add an empty line before `spyOn(...` to separate assignments block and func calls block.